google-services-plugin/build.gradle.kts

```diff
@@ -20,7 +20,7 @@ plugins {
 }
 
 group = "com.google.gms"
-version = "5.0.0"
+version = "4.4.0"
```

Comment:
intended downgrade?

Reply:
yes, 5.0.0 was never released, and since we removed all the behavior changes we released as 4.4.0